docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -110,7 +124,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Every session must have logging turned on. This allows users to have all the logs in case troubleshooting is required. You can find the log files at the location of the 'Filename' shown above. This location contain two log files, \n",
+    "You can find the log files at \".qcodes\" directory, typically located at your home folder (e.g., see the corresponding path to the \"Filename\" key above). This path contains two log files: \n",
```

Comment:
Ali thanks for all the other changes. But why did you removed the line asking people to start logging in each session of the Kernel. I think we can tone down the statement to make it sound like a suggestion instead of command(which it is at the moment) but not completely remove it.

Reply:
I did not remove it, just made it a suggestion and change its position for coherency, see line 96.